Gruntfile.js

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env node
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Generated on 2013-12-04 using generator-backbone-amd 0.0.4
+
+
+// # Globbing
+// for performance reasons we're only matching one level down:
+// 'test/spec/{,*/}*.js'
+// use this if you want to recursively match all subfolders:
+// 'test/spec/**/*.js'
+
+module.exports = function (grunt) {
+  // load all grunt tasks based on environment
+  if (process.env.NODE_ENV && process.env.NODE_ENV === 'production') {
```

Comment:
Nit: this may need some tweaking as well.
According to [lib/config.js](https://github.com/mozilla/fxa-basket-proxy/pull/2/files#diff-f7bf2b665273dfa66ffa4ad7c0a52bb9R13), it looks like the `NODE_ENV` may be "prod", not "production":

```js
var conf = module.exports = convict({
  env: {
    doc: 'The current node.js environment',
    default: 'prod',
    format: [ 'dev', 'test', 'stage', 'prod' ],
    env: 'NODE_ENV'
  },
```

Not sure if this is worth wrapping in a `conf.get('env') === 'prod';` style thing, or maybe we need a better abstraction for determining if we're running in dev/stage/prod environment.